src/x509.cc

```diff
@@ -45,6 +45,85 @@ std::string parse_args(const Nan::FunctionCallbackInfo<v8::Value>& info) {
   return *String::Utf8Value(info[0]->ToString());
 }
 
+
+
+NAN_METHOD(verify) {
+  Nan::HandleScope scope;
+  OpenSSL_add_all_algorithms();
```

Comment:
In fact, they should be able to `require('crypto')` in their own code if they want to use it in a version earlier than 1.1.0 since it calls `OpenSSL_add_all_algorithms` when it's loaded. That would allow them to remove the dependency on their own when they upgrade without us having to change our code.